### PR TITLE
Updated Live loader command

### PR DESCRIPTION
**Pull request by kaushik691**
Applying Fix for live loader issue

### CODE REVIEW

File: lib/dgraph_lib/Dgraph/components/setup_configurations.py [hunk 1]
```diff
@@ -3,7 +3,9 @@
 """
 import json
 import pathlib
+from subprocess import Popen, PIPE
 from robot.api import logger
+import io
```
Comment (souravDgraph) [inline lib/dgraph_lib/Dgraph/components/setup_configurations.py:8]: are we using this ?

File: test_suites/dgraph/Linux/dgraph_suite_01.robot [hunk 2]
```diff
@@ -31,23 +31,23 @@ TC_03 Perform NFS backup and restore data
...
-     Create NFS Backup    increment
-     perform a restore on backup
-
-TC_06 Perform parallel live loads.
-     [Documentation]    Perform live load data.
-     ...    *Author*: Sourav
-     [Tags]    regression
-     Execute Multiple Parallel Live Loader with rdf and schema parameters    ${rdf_file}    ${schema_file}    2
+#
+#TC_04 Perform parallel live and bulk load on data
```
Comment (souravDgraph) [inline test_suites/dgraph/Linux/dgraph_suite_01.robot:35]: These TCs are commented?

**Review by souravDgraph — CHANGES_REQUESTED** [review]

File: test_suites/dgraph/Linux/dgraph_suite_01.robot (hunk 2 above)
Comment (kaushik691) [reply test_suites/dgraph/Linux/dgraph_suite_01.robot:35]: I'll change

File: lib/dgraph_lib/Dgraph/components/setup_configurations.py (hunk 1 above)
Comment (kaushik691) [reply lib/dgraph_lib/Dgraph/components/setup_configurations.py:8]: No.. will update this

**Review by souravDgraph — APPROVED** [review]

**Pull request merged**